### PR TITLE
Bugfix/shared elements

### LINKED ISSUE #24: Dark mode switcher not switching when clicked on light mode side
Dark mode switcher not switching when clicked on light mode side

### LINKED ISSUE #59: Not working avatar after load
Not working awatar after login.  Client/LoginFormComponent.razor Line 64 SetItemAsync insted of SetItemAsStringAsync

### PR DESCRIPTION
**Pull request by Ventan00**
Removed Cascading parameter connected with theme color
Fixed login theme style to match standard mechanism
Fixed Image loading after login 

Close #59 
Close #24

### CODE REVIEW

File: Client/Shared/TopBarComponent.razor [hunk 1]
```diff
@@ -25,31 +27,36 @@
...
     ///     Zmienna przechowująca Awatar
     /// </summary>
     private string Avatar;
 
-
     protected override async Task OnInitializedAsync()
     {
         Avatar = await _localStorageService.GetItemAsync<string>("avatar");
+        _navManager.LocationChanged += MyListener;
+        Console.WriteLine("IA:" + Avatar);
```
Comment (Patrykoss) [inline Client/Shared/TopBarComponent.razor]: Is it necesarry to write to console?

File: Client/Shared/TopBarComponent.razor [hunk 2]
```diff
@@ -25,31 +27,36 @@
...
+    }
+
+    private void MyListener(object? sender, LocationChangedEventArgs e)
+    {
+        Console.WriteLine("called: " + sender + " ; " + e.IsNavigationIntercepted);
```
Comment (Patrykoss) [inline Client/Shared/TopBarComponent.razor]: Is it necesarry to write to console?

**Review by Patrykoss — CHANGES_REQUESTED** [review]

File: Client/Shared/TopBarComponent.razor (hunk 2 above)
Comment (Ventan00) [reply Client/Shared/TopBarComponent.razor]: No it's not. I just forgot to remove it 😄

**Review by Patrykoss — APPROVED** [review]
Now it's ok

File: Client/Components/FormInputComponent.razor [hunk 3]
```diff
@@ -1,16 +1,10 @@
-﻿<div class="input-container @(!IsWhiteTheme ? "input-container-dark" : "")">
-    <InputText @bind-Value="Value" type="@Type" placeholder="@Placeholder" @oninput="OnInputValueChanged" />
+﻿<div class="input-container input-container-dark">
```
Comment (pnarozniak) [inline Client/Components/FormInputComponent.razor]: please remove input-container-dark class

File: Client/Shared/MainLayout.razor [hunk 4]
```diff
@@ -5,18 +5,16 @@
...
 <div class="sidebar">
-    <NavMenuComponent EventCallback="HandleThemeChange" />
+    <NavMenuComponent EventCallback="HandleThemeChange"/>
 </div>
 <div class="page @(isWhiteTheme ? "light-mode" : "dark-mode")">
-    <div class="topbar @(isWhiteTheme ? "" : "topbar-dark")">
-        <CascadingValue Value="isWhiteTheme">
-            <TopBarComponent />
-        </CascadingValue>
+    <div class="topbar @(isWhiteTheme ? "light-mode" : "dark-mode")">
```
Comment (pnarozniak) [inline Client/Shared/MainLayout.razor]: old style for dark-mode should only be applied for (div class="page")
please remove: @(isWhiteTheme ? "light-mode" : "dark-mode") from (div class="topbar")

**Review by pnarozniak — CHANGES_REQUESTED** [review]
Please apply changes mentioned bellow

File: Client/Shared/TopBarComponent.razor [hunk 5]
```diff
@@ -46,12 +46,10 @@
     {
         Avatar = await _localStorageService.GetItemAsync<string>("avatar");
         _navManager.LocationChanged += MyListener;
-        Console.WriteLine("IA:" + Avatar);
     }
 
     private void MyListener(object? sender, LocationChangedEventArgs e)
```
Comment (pnarozniak) [inline Client/Shared/TopBarComponent.razor:57]: Missing docs

**Review by pnarozniak — CHANGES_REQUESTED** [review]

File: Client/Shared/MainLayout.razor (hunk 4 above)
Comment (Ventan00) [reply Client/Shared/MainLayout.razor]: Why you want two different mechanism for same functionality?

Comment (pnarozniak) [reply Client/Shared/MainLayout.razor]: I mean you have to apply old mechanism to global div (page).
If you don't the new mechanism will not work. But you shouldn't use old mechanism in nested divs.

Comment (pnarozniak) [reply Client/Shared/MainLayout.razor]: So it should look like that:

div class="page @(isWhiteTheme ? "light-mode" : "dark-mode")"
    div class="topbar"

Comment (pnarozniak) [reply Client/Shared/MainLayout.razor]: As i saw in css, you are not even using the mechanism you left in code so

Comment (pnarozniak) [reply Client/Shared/MainLayout.razor]: .css has a new version
.razor has an old one

.light-mode .topbar {
    background: white;
    box-shadow: -3px -13px 15px 13px #ccc;
}

.dark-mode .topbar {
    background: #3F4D54;
    box-shadow: -3px -13px 15px 13px #333F44;
}

**Review by pnarozniak — APPROVED** [review]

**Pull request merged**